cli/schema/cypress.schema.json

```diff
@@ -32,6 +32,11 @@
       "default": null,
       "description": "Port used to host Cypress. Normally this is a randomly generated port"
     },
+    "quiet": {
+      "type": "boolean",
+      "default": false,
+      "description": "Whether Cypress will run quietly, using only the configured reporter"
+    },
```

Comment:
`cypress.schema.json` is for adding new config options, so since we're just adding a CLI flag/module option, it doesn't need to be here

```suggestion
```